src/CppUTest/CommandLineTestRunner.cpp

```diff
@@ -145,8 +153,11 @@ bool CommandLineTestRunner::parseArguments(TestPlugin* plugin)
     return false;
   }
 
-  if (arguments_->isJUnitOutput())
+  if (arguments_->isJUnitOutput()) {
     output_= createJUnitOutput(arguments_->getPackageName());
+    if (arguments_->isVerbose())
+      output_ = createCompositeOutput(output_, createConsoleOutput());
+  }
   else
     output_ = createConsoleOutput();
   return true;
```

Comment:
I guess I dont like the new which is not balanced by a delete.
I also don't really like that a different output is created depending on the arguments.
My solution required no changes to CommandLineTestRunner....
In isolation, I like your CompositeTestOutput.
However, I do feel that my solution integrates more nicely into the existing code.

I there some way that you can avoid new'ing in one place and delete'ing in another?